### ISSUE TITLE
Minor Bug - Telegram does not change between different sound outputs / inputs on Linux when changed

### ISSUE THREAD

**Issue by luisalvarado** [telegramdesktop/tdesktop#928]
Using Telegram, if am for example hearing through my headphones a recorded audio and I want to change it to the speakers, On Ubuntu I go to the audio settings and select from the list the Speakers (Which would deselect the headphones automatically), but Telegram does not get notified about the change. I have to close the Telegram app and open it again for the audio to come out through the speakers.

So basically the audio input/output is read only on initialization and there is no event system that checks if the sound input/output was changed.

**Comment by auchri** [comment]
Look at #869 #418

**Comment by luisalvarado** [comment]
Thank you auchri. Will close as duplicate.
